### DISCUSSION TITLE
Mathematical Formula Editing

### DISCUSSION THREAD

**Discussion by lair001** [shuzijun/markdown-editor#59] [Q&A]
Does the editor provide assistance with writing mathematical formulas in *.md files?

**Comment by shuzijun** [comment]
Refer: https://katex.org/docs/supported.html

E.g
```
$$
\frac{1}{
  \Bigl(\sqrt{\phi \sqrt{5}}-\phi\Bigr) e^{
  \frac25 \pi}} = 1+\frac{e^{-2\pi}} {1+\frac{e^{-4\pi}} {
    1+\frac{e^{-6\pi}}
    {1+\frac{e^{-8\pi}}{1+\cdots}}
  }
}
$$
```

**Reply by lair001** [reply]
That's not what I'm asking. I was wondering if the gui provided assistance with writing mathematical formulas like there is for adding, e.g, tables, or if there are plans to add such assistance in the future.  I don't use LaTeX often enough to commit the commands to memory.
